src/components/Accordions/Accordion.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import { Accordion, AccordionItem } from '@nextui-org/accordion';
+
+interface AccordionItemProps {
+  title: string;
+  iconSrc?: string;
+  content: React.ReactNode | string;
+}
+
+interface AccordionsProps {
+  items: AccordionItemProps[];
+}
+
+export default function Accordions({ items }: AccordionsProps) {
+  return (
+    <div className="accordion-wrapper">
+      <Accordion variant="splitted" className="accordion-container">
```

Comment:
could you add some padding between the info icon a the content
![image](https://github.com/user-attachments/assets/2fd30f7e-842d-4fab-9cac-c941e622c9d8)